### PR TITLE
Finalising GraphQL documentation for launch

### PR DESCRIPTION
**Pull request by KateVomBerg**

### CODE REVIEW

File: api-docs/graphql-api-rate-limits [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+The rate limiting rules of our GraphQL API are connected to the calculated complexity scores of your queries. 
+A GraphQL query can be arbitrarily complex, therefore we parse all incoming queries, calculate their complexity scores in points and subtract those from the rate limit for your API user.
+
+**Currently the Rate Limits for our GraphQL API are:**
+
+- A single query is limited to 50000 complexity points.)
+- All queries within a 10 minute period are limited to 500000 complexity points per API user.
+
+
+> TODO
```
Comment (KateVomBerg) [inline api-docs/graphql-api-rate-limits:24]: @mmt-trackunit Can you provide a short description of how the gql api responds if a rate limit is hit?

File: api-docs/graphql-api-introduction.md [hunk 2]
```diff
@@ -51,3 +51,16 @@ The codeword differs for each preview field and you can find it by looking in th
...
 
 Additionally you can also download the [Public schema](https://apps.iris.trackunit.com/graphql-public-viewer/schema.gql) of the Trackunit Iris GraphQL API.
+
+## Advise for efficient querying with GraphQL
+
+When querying a GraphQL API, adopting certain practices can significantly enhance efficiency and responsiveness: 
+
+- Begin by precisely defining your data requirements using field selection, requesting only the information you genuinely need to reduce unnecessary data transfer. 
+- Leverage query fragments to reuse common selections across multiple queries, avoiding redundant code and promoting consistency. 
+- Utilize pagination techniques for fetching large sets of data, such as utilizing the `first`, `last`, `before`, and `after` arguments to manage the amount of data returned in each query.
```
Comment (KateVomBerg) [inline api-docs/graphql-api-introduction.md:61]: @mmt-trackunit Did we end up making 'first' or 'last' required?

File: api-docs/graphql-api-elasticsearch [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+category: 652e3a8e279c3f001f9acdc3
+---
+
+In the Trackunit Iris platform we have integrated the Trackunit Search Engine powered by Elasticsearch with our GraphQL API. 
+This provides a powerful way for you to enhance your query capabilities. 
+By utilizing Elasticsearch as a data source within the GraphQL ecosystem, you can efficiently filter and retrieve the exact data from your fleet that you need. 
+
+When you initiate a GraphQL query, you can leverage Elasticsearch's robust querying capabilities, including full-text search, filtering, sorting, and aggregation. 
+By structuring your GraphQL queries with relevant Elasticsearch filters, such as range queries, term queries, or boolean queries, you can precisely narrow down the data to be retrieved.
+
```
Comment (KateVomBerg) [inline api-docs/graphql-api-elasticsearch:12]: @mmt-trackunit I would really like to add an image here, but I don't know how to do this via github 😂  Would you mind adding the picture that is in this paper doc? https://www.dropbox.com/scl/fi/e6tipd1vinuzcmj0pc2wk/Product-Launch-Brief-PUBLIC-GraphQL-API.paper?rlkey=qvzq4g5156cwdedxysosn3fll&dl=0#:h2=Search-&-Filter-with-the-Track

File: api-docs/graphql-api-introduction.md (hunk 2 above)
Comment (mmt-trackunit) [reply api-docs/graphql-api-introduction.md:61]: Yes. But some queries takes a list of IDs as well, and when they are given a list of IDs `first` or `last` is not required.

File: api-docs/graphql-api-elasticsearch (hunk 3 above)
Comment (mmt-trackunit) [reply api-docs/graphql-api-elasticsearch]: Added it now

File: api-docs/graphql-api-vs-rest.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+2. Efficient Data Retrieval: In scenarios where bandwidth is a concern (e.g., mobile apps), GraphQL's ability to retrieve all required data in a single request can reduce the number of round-trips between the client and server.
+3. Complex Relationships: When dealing with complex relationships between entities, GraphQL's nested queries and efficient data loading can simplify data fetching on the client side.
+4. Avoiding Over-Fetching and Under-Fetching: GraphQL allows clients to request exactly the data they need, reducing the chances of over-fetching (retrieving more data than necessary) or under-fetching (not getting enough data).
+5. Aggregating Data: GraphQL can aggregate data from multiple sources and present it as a single unified API, making it a good choice for applications that pull data from different services.
+
+In summary, use REST APIs when simplicity and compatibility with a variety of clients are your main concerns. 
+Use GraphQL APIs when you need more flexibility in data retrieval, have complex data relationships, and want to optimize data fetching for e.g. app development.
```
Comment (mmt-trackunit) [inline api-docs/graphql-api-vs-rest.md:31]:
```suggestion
Use GraphQL APIs when you need more flexibility in data retrieval, retrieve complex data relationships, and want to optimize data fetching for e.g. app development.
```

**Review by mmt-trackunit — APPROVED** [review]
Nice docs 🥇
I've added a few bits and pieces the places you tagged me.
The image will not work until it is merged.

**Pull request merged**